types/raidgroupconfig_test.go

```diff
@@ -233,6 +233,13 @@ func TestValidate(t *testing.T) {
 			},
 			isErr: true,
 		},
+		"stripe pool and device count is less than min count": {
+			src: &RaidGroupConfig{
+				Type:             PoolRAIDTypeStripe,
+				GroupDeviceCount: 0,
```

Comment:
Can you put some UT with negative values as well

Reply:
@AmitKumarDas  UT with negative values is already present.

Reply:
awesome @shovanmaity